Author: fs
Date: Mon Jan  4 12:56:14 2010
New Revision: 277843

Log:
#i107917# do not access xDataSource for reading, but use getDataSource()

Modified:
   cws/dba32k/wizards/com/sun/star/wizards/db/DBMetaData.java

Modified: cws/dba32k/wizards/com/sun/star/wizards/db/DBMetaData.java
==============================================================================
--- wizards/com/sun/star/wizards/db/DBMetaData.java	Mon Jan  4 12:36:09 2010	(r277842)
+++ wizards/com/sun/star/wizards/db/DBMetaData.java	Mon Jan  4 12:56:14 2010	(r277843)
@@ -599,8 +599,8 @@
 
     public void getDataSourceInterfaces() throws Exception
     {
-        xCompleted = (XCompletedConnection) UnoRuntime.queryInterface(XCompletedConnection.class, xDataSource);
-        xDataSourcePropertySet = (XPropertySet) UnoRuntime.queryInterface(XPropertySet.class, xDataSource);
+        xCompleted = (XCompletedConnection) UnoRuntime.queryInterface(XCompletedConnection.class, getDataSource());
+        xDataSourcePropertySet = (XPropertySet) UnoRuntime.queryInterface(XPropertySet.class, getDataSource());
         bPasswordIsRequired = ((Boolean) xDataSourcePropertySet.getPropertyValue("IsPasswordRequired")).booleanValue();
     }
 
